homeassistant/components/owntracks/device_tracker.py

```diff
@@ -154,4 +154,5 @@ async def async_added_to_hass(self):
     def update_data(self, data):
         """Mark the device as seen."""
         self._data = data
-        self.async_write_ha_state()
+        if self.hass:
```

Comment:
We should probably use the dispatch helper instead to connect the owntracks context to the entity update callback and and connect it in async_added_to_hass. This is our standard for GPS entities.

How does disabling owntracks entities work?

Reply:
It will work because it will keep hitting this guard and never write.

You're right in that it should be updated to dispatcher. However there is no code owner for OwnTracks.

Reply:
I was wrong. But after #36742 it will work as we expect.